src/ILLink.Shared/DiagnosticString.cs

```diff
@@ -8,24 +8,24 @@
 		public DiagnosticString (DiagnosticId diagnosticId)
 		{
 			var resourceManager = SharedStrings.ResourceManager;
-			_titleFormat = resourceManager.GetString ($"{diagnosticId}Title");
-			_messageFormat = resourceManager.GetString ($"{diagnosticId}Message");
+			_titleFormat = resourceManager.GetString ($"{diagnosticId}Title") ?? string.Empty;
```

Comment:
Note that not all message strings have a title counterpart, take for example `BaseRequiresMismatchMessage` (there's no BRMTitle string).

Reply:
Never mind - looks like not all diagnostics have fixed messages, because we use different strings depending on the context - see here:
https://github.com/dotnet/linker/blob/3418bcd1d864ce7f62d789c8961f9c2d750fb079/src/ILLink.Shared/MessageFormat.cs#L23-L33